app/components/views/GetStartedPage/messages/messages.jsx

```diff
@@ -58,6 +59,10 @@ export const messages = defineMessages({
   passphrasePlaceholder: {
     id: "getStarted.discoverAddresses.passphrasePlaceholder",
     defaultMessage: "Private Passphrase"
+  },
+  confirmSeedMissingWordError: {
+    id: "getStarted.confirmSeed.missingWordError",
+    defaultMessage: "*Please confirm the missing words"
```

Comment:
I know this is the original message, but re-testing, this error reads weird when you have verified all seed words, but one of them is wrong. There are no "missing" words, but there _is_ a "wrong word". If possible, I suggest differentiating between not having entered all words yet and an actually wrong word.

Another thing I've noticed is that there's now no feedback on _which_ word is the wrong one. Is this intentional?

Reply:
@matheusd, thanks for the review.
The error message differentiating is a good idea. I've added it.

I've left out the wrong word feedback intentionally, but I'm not sure about it. Since the goal of this confirmation view is to ensure the user wrote down the correct seed words, I feel the error messages give too much room for being lazy and easy to click through this view by trial and error.

I've added the error feedback though. It is easier to see this during operation. 
It is not in design, and I choose the colors. @MariaPleshkova, please take a look at the screenshot of it.

Reply:
> I feel the error messages give too much room for being lazy and easy to click through this view by trial and error.

Indeed, displaying the error directly upon clicking seems too easy to cause the user to be lazy.

Another idea would be to only do the seed validation after the user fills all words, and _then_ display whether a particular word is wrong or not.

Reply:
Good idea 👍️

Reply:
Agree with @matheusd regarding the seed validation after the user fills all words. And the red colored wrong words are also look good. I'd also suggest to show maybe an error message after validation as well? Like "you have chosen the wrong words" or "some words from the seed are incorrect" or something like that. And a user can be automatically moved up to the top of the page after validation and see this message. 
An example:
![image](https://user-images.githubusercontent.com/8841018/142618458-e07cab62-46ad-4d09-bea4-2e4097e05676.png)

Reply:
@MariaPleshkova There was an error message on the bottom but the screenshot did not contain it.  I've updated it following your suggestion and uploaded a fresh screenshot to the description. Thank you